Api/Services/RestaurantService.cs

```diff
@@ -1606,5 +1607,61 @@ public async Task<Result<List<AvailableHoursVM>>> GetAvailableHoursAsync(int res
 
             return mergedAvailableHours;
         }
+
+        /// <summary>
+        /// Returns tables for the given restaurant
+        /// </summary>
+        /// <param name="restaurantId"></param>
+        /// <param name="user"></param>
+        /// <returns></returns>
+
+        [ErrorCode(nameof(restaurantId), ErrorCodes.NotFound)]
+        public async Task<Result<List<RestaurantTableVM>>> GetTablesAsync(int restaurantId, User user)
+        {
+            var restaurant = await context.Restaurants
+                .AsNoTracking()
+                .Include(r => r.Tables)
+                .OnlyActiveRestaurants()
+                .FirstOrDefaultAsync(r => r.RestaurantId == restaurantId);
+
+            if (restaurant == null)
+            {
+                return new ValidationFailure
+                {
+                    PropertyName = nameof(restaurantId),
+                    ErrorMessage = $"Restaurant with ID {restaurantId} not found",
+                    ErrorCode = ErrorCodes.NotFound
+                };
+            }
+
+            var res = await authorizationService.VerifyRestaurantHallAccess(restaurantId, user.Id);
+
+            if (res.IsError)
+            {
+                return res.Errors;
+            }
+
+            var tables = restaurant.Tables
+                .Select(t => new RestaurantTableVM
+                {
+                    TableId = t.TableId,
+                    Capacity = t.Capacity,
+                    Status = TableStatus.Availiable, 
```

Comment:
mówiłeś że potrzebujecie VisitId, piszę na wszelki wypadek

Reply:
usunąłem status, dodałem visitId, to czy jest zajęty stolik, wystarczy nam sprawdzenie czy visitId == null